src/sentry/static/sentry/app/views/organizationEvents/utils/canIncludePreviousPeriod.jsx

```diff
@@ -0,0 +1,16 @@
+import {parsePeriodToHours} from 'app/utils';
+// Max period (in hours) before we can no long include previous period
+const MAX_PERIOD_HOURS_INCLUDE_PREVIOUS = 45 * 24;
```

Comment:
Can we do like retention/2 instead of hardcoding the 45 as retention varies by plan?